src/model/model.ts

```diff
@@ -1,43 +1,106 @@
-import { getCollection, validateSchema } from '..';
-import { save } from '../handler/save';
-import { find } from '../handler/find';
 import { generateUUID } from '../utils/generate-uuid';
+import { extractDataFromModel } from '../utils/extract-data-from-model';
+import { remove } from '../handler/remove';
```

Comment:
Are we using this?

Reply:
Fixed: all unused variables were removed